Assets/HoloToolkit/Utilities/Scripts/WindowsApiChecker.cs

```diff
@@ -13,8 +13,10 @@ public static class WindowsApiChecker
         static WindowsApiChecker()
         {
 #if !UNITY_EDITOR && UNITY_WSA
+#if UNITY_2017_2_OR_NEWER
```

Comment:
Do these calls need to be `#if`ed out? It seems like they'd just return false on earlier build targets.